README.rst

```diff
@@ -472,7 +472,7 @@ template ``context_processors`` in ``settings.py`` as shown below.
 ``register_menu_group``
 ^^^^^^^^^^^^^^^^^^^^^^^
 
-It allows us to register different menu item or group at different position in the Main Navigation Menu.
+Allows registering a new menu item or group at the specified position in the Main Navigation Menu.
```

Comment:
I have made this change to make it consistent with https://github.com/openwisp/openwisp-utils#register_dashboard_template